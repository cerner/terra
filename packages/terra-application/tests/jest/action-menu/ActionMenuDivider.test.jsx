import React from 'react';

import ActionMenuDivider from '../../../src/action-menu/ActionMenuDivider';

describe('ActionMenuDivider', () => {
  it('should render with minimal props', () => {
    const wrapper = shallow((
      <ActionMenuDivider />
    ));
    expect(wrapper).toMatchSnapshot();
  });
});
