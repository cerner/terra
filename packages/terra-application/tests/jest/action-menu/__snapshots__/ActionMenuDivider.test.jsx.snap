// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionMenuDivider should render with minimal props 1`] = `
<li
  aria-hidden={true}
  className="action-menu-divider"
>
  <div
    className="line"
  />
</li>
`;
